test/e2e/conditions.go

```diff
@@ -26,17 +29,28 @@ func conditionsToYaml(conditions nmstatev1alpha1.ConditionList) string {
 	return string(manifest)
 }
 
-func enactmentConditionsStatus(node string) nmstatev1alpha1.ConditionList {
-	key := types.NamespacedName{Name: TestPolicy}
-	policy := nodeNetworkConfigurationPolicy(key)
+func nodeNetworkConfigurationEnactment(key types.NamespacedName) nmstatev1alpha1.NodeNetworkConfigurationEnactment {
+	state := nmstatev1alpha1.NodeNetworkConfigurationEnactment{}
+	Eventually(func() error {
+		return framework.Global.Client.Get(context.TODO(), key, &state)
+	}, ReadTimeout, ReadInterval).ShouldNot(HaveOccurred())
+	return state
+}
+
+func enactmentKey(node string, policy string) types.NamespacedName {
+	return types.NamespacedName{Name: node + "-" + policy}
```

Comment:
please try to use a different separator as we talked about

Reply:
It's done